integrations/autotools/building_with_autotools.rst

```diff
@@ -0,0 +1,318 @@
+.. _building_with_autotools:
+
+|gnu_logo|
+
+Building with Autotools
+=======================
+
+If you are using **configure**/**make** you can use **AutoToolsBuildEnvironment** helper.
+This helper sets ``LIBS``, ``LDFLAGS``, ``CFLAGS``, ``CXXFLAGS`` and ``CPPFLAGS`` environment variables based on your requirements.
+
+.. code-block:: python
+   :emphasize-lines: 13, 14, 15
+
+   from conans import ConanFile, AutoToolsBuildEnvironment
+
+   class ExampleConan(ConanFile):
+      settings = "os", "compiler", "build_type", "arch"
+      requires = "poco/1.9.4"
+      default_options = {"poco:shared": True, "openssl:shared": True}
+
+      def imports(self):
+         self.copy("*.dll", dst="bin", src="bin")
+         self.copy("*.dylib*", dst="bin", src="lib")
+
+      def build(self):
+         autotools = AutoToolsBuildEnvironment(self)
```

Comment:
Legacy integration, needs to be migrated.

Reply:
I know, this is just a draft PR yet